scripts/get-github-release.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
Have you considered to write it in Python? The other script is also in Python and it would help to keep the scripts consistent. Also, I think the same outcome could be achieved in a simpler way using Python. Maybe this script could be used as reference: https://github.com/ComplianceAsCode/content/blob/master/utils/release_helper.py#L264-L272